OctoKit/Issue.swift

```diff
@@ -81,12 +87,13 @@ public extension Octokit {
     /**
      Fetches the issues of the authenticated user
      - parameter session: RequestKitURLSession, defaults to NSURLSession.sharedSession()
+     - parameter state: Issue state. Defaults to open if not specified.
      - parameter page: Current page for issue pagination. `1` by default.
      - parameter perPage: Number of issues per page. `100` by default.
      - parameter completion: Callback for the outcome of the fetch.
      */
-    public func myIssues(_ session: RequestKitURLSession = URLSession.shared, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Issue]>) -> Void) -> URLSessionDataTaskProtocol? {
-        let router = IssueRouter.readAuthenticatedIssues(configuration, page, perPage)
+    public func myIssues(_ session: RequestKitURLSession = URLSession.shared, state: State? = nil, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Issue]>) -> Void) -> URLSessionDataTaskProtocol? {
```

Comment:
Instead of setting the default `state` to `nil` why not setting it to `Open` and make it non-optional? You are setting the state to open anyway [here](https://github.com/nerdishbynature/octokit.swift/pull/56/files#diff-d1085b98387def82469a5923f3f069bdR246).